core/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -155,6 +157,10 @@ private TachyonFS(InetSocketAddress masterAddress, boolean zookeeperMode) throws
     mMasterClient =
         mCloser.register(new MasterClient(mMasterAddress, mZookeeperMode, mExecutorService));
     mWorkerClient = mCloser.register(new WorkerClient(mMasterClient, mExecutorService));
+
+    String scheme = mZookeeperMode ? Constants.SCHEME_FT : Constants.SCHEME;
+    String authority = mMasterAddress.getHostName() + ":" + mMasterAddress.getPort();
```

Comment:
If `mMasterAddress.toString()` returns "/127.0.0.1:19998", after calling `mMasterAddress.getHostName()` in this statement, the return value will become "localhost/127.0.0.1:19998", this will affect `TachyonFSTest:toStringTest`, so these two commits have to be submitted in the same PR, otherwise, jenkins will fail.